node/src/components/consensus/highway_core/state/index_panorama.rs

```diff
@@ -14,19 +14,20 @@ use crate::components::consensus::{
 pub(crate) type IndexPanorama = ValidatorMap<IndexObservation>;
 
 /// The observed behavior of a validator at some point in time.
-#[derive(Clone, DataSize, Eq, PartialEq, Serialize, Deserialize, Hash)]
+#[derive(Clone, Copy, DataSize, Eq, PartialEq, Serialize, Deserialize, Hash)]
 pub(crate) enum IndexObservation {
-    None,
+    /// We have evidence that the validator is faulty.
     Faulty,
-    Correct(u64),
+    /// We have that number of units from the validator.
+    /// That is also the sequence number of the first unit we are still missing.
+    Count(u64),
```

Comment:
How do you feel about renaming it to `HighestSeq` or something similar?

Reply:
But it isn't! It's one more than the highest sequence number. Maybe `LowestMissingSeq`, or `UnitCount`?

Reply:
`LowestMissing` sounds good. It feels more natural to talk in terms of the missing sequence numbers than making a correlation between count and what should be requested.

Reply:
As discussed, I'll call it `NextSeq`.

Reply:
Done in https://github.com/casper-network/casper-node/pull/2167/commits/e6e4cf71e3ba9ac9b6569a3c9023584c76415f75.